rust/headless-client/src/dns_control/linux.rs

```diff
@@ -4,45 +4,54 @@ use std::{net::IpAddr, process::Command, str::FromStr};
 
 mod etc_resolv_conf;
 
-const FIREZONE_DNS_CONTROL: &str = "FIREZONE_DNS_CONTROL";
-
-pub fn system_resolvers_for_gui() -> Result<Vec<IpAddr>> {
-    get_system_default_resolvers_systemd_resolved()
-}
-
-#[derive(Clone, Debug)]
-enum DnsControlMethod {
+/// Methods to control the system's DNS resolution
+#[derive(clap::ValueEnum, Clone, Debug)]
+pub(crate) enum Method {
+    /// Explicitly disable DNS control
+    ///
+    /// This is for avoiding surprise if a user runs the Client and it implicitly
+    /// doesn't control DNS. The user must tell the headless Client how to control DNS.
+    ///
+    /// We don't use an `Option<Method>` because then `clap` may treat the arg as
+    /// optional.
+    Disabled,
     /// Back up `/etc/resolv.conf` and replace it with our own
     ///
     /// Only suitable for the Alpine CI containers and maybe something like an
     /// embedded system
     EtcResolvConf,
-    /// Cooperate with NetworkManager (TODO)
-    NetworkManager,
```

Comment:
We didn't get around to NetworkManager yet, so I'm removing it for the time being